src/components/index.jsx

```diff
@@ -1,12 +1,15 @@
 import React, { Component } from 'react';
 import Welcome from './Welcome.jsx';
-
+import {Provider} from 'react-redux';
```

Comment:
Please rename this file to App.jsx as said yesterday. Also, import the provider and store in src/index.js where the App component is called

Reply:
I finished implementing this.